pkg/gds/certs.go

```diff
@@ -143,20 +143,17 @@ func (s *Service) submitCertificateRequest(r *models.CertificateRequest) (err er
 		return fmt.Errorf("could not retrieve pkcs12password: %s", err)
 	}
 
-	params := make(map[string]string)
-	params["commonName"] = r.CommonName
-	params["dNSName"] = r.CommonName
-	params["pkcs12Password"] = string(pkcs12Password)
+	var params map[string]string
 
 	profile := s.certs.Profile()
 	if profile == sectigo.ProfileCipherTraceEndEntityCertificate || profile == sectigo.ProfileIDCipherTraceEndEntityCertificate {
-		// Default to TRISA Production locality since none has been provided.
-		// TODO: make this part of the certificate request (See SC-2606).
-		params["organizationName"] = "TRISA Production"
-		params["localityName"] = "Menlo Park"
-		params["stateOrProvinceName"] = "California"
-		params["countryName"] = "US"
+		params = r.Params
+	} else {
+		params = make(map[string]string)
 	}
+	params["commonName"] = r.CommonName
+	params["dNSName"] = r.CommonName
+	params["pkcs12Password"] = string(pkcs12Password)
 
```

Comment:
Also much neater! 

Do we have tests related to cert manager? I'd like to create a story that uses the sectigo mock to ensure that the params are created correctly for both `ProfileCipherTraceEndEntityCertificate` (includes all the subject information + common name, dns name, password) and for `ProfileCipherTraceEE` (just common name, dns name, password). If we don't have tests related to cert manager, I guess it will be two stories - one to create test stubs for cert manager and the other to implement the test above.

Reply:
We do have tests for the cert manager but I don't the Sectigo mock is checking the parameters we're building here, so I think that would be a good thing to add to validate this code.

Reply:
Excellent! Would you mind creating a story for it?